### ISSUE TITLE
日志：大佬支持 tfevent 输出效果？

### ISSUE THREAD

**Issue by EdwardChan5000** [JayYip/bert-multitask-learning#118]

**Comment by JayYip** [comment]
输出效果指的是？

**Comment by EdwardChan5000** [comment]
简单想把 输出print出来的 acc 和 loss 这些用 tfboard 来看

**Comment by JayYip** [comment]
这个貌似有吧？

**Comment by EdwardChan5000** [comment]
喔+ 有了吗? 是有个存  tfevent 的超参 or 地方?

**Comment by JayYip** [comment]
确认了下, 没有加上`tf.keras.callbacks.TensorBoard` callback. 在调用`train_bert_multitask`函数的时候传一个`callbacks=[tf.keras.callbacks.TensorBoard(log_dir=log_path)]`参数试试
